### DISCUSSION TITLE
Shop mod

### DISCUSSION THREAD

**Discussion by priestkos** [Glazelf/RiseHax#8] [Ideas]
A possible implementation of the item shop mod that gives access to all non material items eg. Lifepowder, Max Potion etc. to be available for purchase from the shop. Could scale the prices for balancing purposes if needed.

Addendum: Once the game releases I can list items that cannot be bought normally and suggest prices for them in store.

**Comment by Glazelf** [comment]
In World you couldn't edit the shop in save edit since it isn't saved.

It might be possible to add it in memory editing, but it would revert anyways if you restart the game. And you can just edit items into your inventory.

For specifically permanent shop editing you will likely need mods again.

**Reply by enesyue** [reply]
didnt want to create a new discussion due to not exactly knowing how the emulators and such work, so ill just write a reply:

is this usable with emulators such as ryujinx?

**Reply by Glazelf** [reply]
Should've made a new discussion on this since it's a new topic.
But; I don't know, I haven't used any Switch emulators. Save editing should obviously be the same, but if Sysbotbase works for it then live editing should too for this.
